contracts/epochs/FlowDKG.cdc

```diff
@@ -202,6 +202,21 @@ pub contract FlowDKG {
     /// The Admin resource provides the ability to begin and end voting for an epoch
     pub resource Admin {
 
+        /// Sets the optional safe DKG success threshold
+        /// Set the threshold to an optional if it isn't needed
+        pub fun setSafeSuccessThreshold(newThresholdPercentage: UFix64?) {
+            pre {
+                !FlowDKG.dkgEnabled: "Cannot set the dkg success threshold while the DKG is enabled"
+                newThresholdPercentage == nil || (newThresholdPercentage! > 0.5 && newThresholdPercentage! < 1.0): "The threshold percentage must be between 0.5 and 1.0"
```

Comment:
I don't think we should be checking for `> 0.5` here. `>= 0` is enough. The only condition is that the final contract threshold is larger than the native threshold (guaranteed by `getSafeSuccessThreshold`). 

Please note the value `0.5` is only a choice we made for this implementation and should only be reflected inside `getNativeSuccessThreshold`. It can be made larger or smaller if we want to. 
Moreover, I only use `0.5` in discussions for simplicity, but the constant is slightly different. `0.5` should not be used in code or formal documentation.

Reply:
@tarakby I've added some comments to the getter methods that hopefully make this clearer